aviary/utils/Fortran_to_Aviary.py

```diff
@@ -313,10 +314,13 @@ def generate_aviary_names(code_bases):
     for code_base in code_bases:
         alternate_names[code_base] = {}
         for key in _MetaData.keys():
-            alt_name = _MetaData[key]['historical_name'][code_base]
-            if isinstance(alt_name, str):
-                alt_name = [alt_name]
-            alternate_names[code_base][key] = alt_name
+            historical_dict = _MetaData[key]['historical_name']
+            if historical_dict:
+                if code_base in historical_dict:
```

Comment:
I think this can be combined into `if historical_dict and code_base in historical_dict:`